pulsar-client-cpp/tests/AuthPluginTest.cc

```diff
@@ -190,14 +191,15 @@ TEST(AuthPluginTest, testTlsDetectHttpsWithHostNameValidation) {
 
 namespace testAthenz {
 std::string principalToken;
-void mockZTS(int port) {
+void mockZTS(Latch* latch, int port) {
```

Comment:
Passing pointers around can result ambiguous on on the ownership of the object. We should use a reference instead: 
```suggestion
void mockZTS(Latch& latch, int port) {
```